core/common/src/test/java/alluxio/conf/ConfigurationBuilder.java

```diff
@@ -24,7 +24,7 @@ public class ConfigurationBuilder {
    * @return the updated configuration builder
    */
   public ConfigurationBuilder setProperty(PropertyKey key, Object value) {
-    mProperties.put(key, value.toString(), Source.RUNTIME);
+    mProperties.put(key, value, Source.RUNTIME);
```

Comment:
why is this change needed?

Reply:
If not, the relevant calls in PollingMasterInquireClientTest will throw the following exception.
`java.lang.ClassCastException: java.lang.String cannot be cast to java.lang.Boolean`